src/Tribe/Date_Utils.php

```diff
@@ -188,7 +188,8 @@ public static function date_only( $date, $isTimestamp = false, $format = null )
 		 * @return string The time only in DB format.
 		 */
 		public static function time_only( $date ) {
-			return date( self::DBTIMEFORMAT, strtotime( $date ) );
+			$date = is_numeric( $date ) ? $date : strtotime( $date );
+			return date( self::DBTIMEFORMAT, $date );
```

Comment:
I realize this isn't new as of this change, but we return the _time_ here and the function is indeed named `time_only()` - yet the opening line of the docblock states `Returns the date only`